### PR TITLE
quickstart: check that internal LB is never used with google-managed cets

### PR DESCRIPTION
**Pull request by danistrebel**
## Description
What's changed, or what was fixed?

- Safeguard to ensure internal load balancer isn't used with google-managed certificate issuer
- Use custom network on the ILB

**Fixes:** #483 

## Housekeeping
(please check all that apply [X], do not edit the text)
- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.
- [ ] PR requires full pipeline run (Run for changes only by default).

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

File: tools/hybrid-quickstart/steps.sh [hunk 1]
```diff
@@ -43,6 +43,11 @@ set_config_params() {
     export CERT_TYPE=${CERT_TYPE:-google-managed}
     echo "- TLS Certificate $CERT_TYPE"
 
+    if [ "$CERT_TYPE" == "google-managed" ] && [ "$INGRESS_TYPE" != "external" ]; then
+        echo "Google Managed Certificates can only be used with an external ingress. SET CERT_TYPE to 'self-signed' (for the scrit to create one for you) or 'skip' if you want to provide your own as a k8s secret later."
```
Comment (kurtkanaskie) [inline tools/hybrid-quickstart/steps.sh:47]: Typo: for the **scrit**

**Comment by kurtkanaskie** [comment]
Tested and works fine, LBTM!

**Review by OmidTahouri — APPROVED** [review]
LGTM

**Pull request merged**